### ISSUE TITLE
ISP blocked raw.github in India

### ISSUE THREAD

**Issue by Pyrole-2002** [LunarVim/LunarVim#4220]
### Problem description

I am in India and Jio and Airtel have blocked raw.github due to which on runnin the installation script I get no output and the script just times out and ends.

Suggesting to produce an output in such cases where the file cant be accessed and maybe the script could automatically find another way to fetch the installer in such cases.

### LunarVim version

NA

### Neovim version (>= 0.9.0)

0.9

### Terminal name

Kitty

### Operating system/version

Manjaro Linux x86_64

**Comment by kobzar** [comment]
Another way - try to use free VPN servers like all world

**Comment by brneor** [comment]
How the script would do such things if the script itself it's not being downloaded? 🤔

**Comment by kobzar** [comment]
Its no LunarVim issue

**Comment by LostNeophyte** [comment]
The command on the website should output something in this case, it currently does nothing

**Comment by kobzar** [comment]
> The command on the website should output something in this case, it currently does nothing

Disable the internet connection on your PC and run the script.
Dont forget to show output

**Comment by LostNeophyte** [comment]
[this ](https://github.com/LunarVim/LunarVim/issues/4020#issuecomment-1502097266)is how it looks, sometimes immediately, sometimes after a few minutes.
we should lower the timeout and remove the --silent flag

**Comment by kobzar** [comment]
And what?
This is LunarVIm issue or ISP in India?
(I mean this issue around of this repo)

**Comment by LostNeophyte** [comment]
both

**Comment by kobzar** [comment]
Third. My car is broken. But i am use lvim. Please help me.

**Comment by LostNeophyte** [comment]
so if a plugin manager showed no output when cloning the plugins and you didn't have access to the repository you wouldn't expect them to show an error?

**Comment by kobzar** [comment]
i sad this is no Lvim issue

**Comment by opalmay** [comment]
> The command on the website should output something in this case, it currently does nothing

I agree, making the output more verbose in case of connection issues is within the scope of lunarvim

**Comment by MDSADABWASIM** [comment]
@Pyrole-2002  It's not an issue of Lunarvim, but a quick workaround that I used is to use Cloudfare DNS resolver
here's a detailed answer related to this: https://stackoverflow.com/a/77045696/8423838

**Comment by wxguy** [comment]
I too faced this issue. I feel that even if it is not directly related to LunarVim, if the issue affecting the LunarVim users, it should be documented atleast in the concern section of lvim official website. This is very important as it took lot of time for me to to debug what went wrong and till then I could not try LunarVim.
